libse/SubtitleFormats/Utx.cs

```diff
@@ -53,37 +56,37 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
         {
             _errorCount = 0;
             subtitle.Paragraphs.Clear();
-            var text = new StringBuilder();
+            var sb = new StringBuilder();
+            char[] splitChars = { '#', ',' };
+            char[] splitChars2 = { ':', '.' };
             for (int i = 0; i < lines.Count; i++)
             {
                 string line = lines[i].Trim();
-
-                if (line.StartsWith('#'))

```

Comment:
Here is where the bug was present. Since subtitle text can start with `#` we should always check if it's time-code or not, if we assume that it's time-code and time-code parsing process fails the text will be ignored.